packages/core/src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -4,7 +4,7 @@
   position: sticky;
   top: 0;
   z-index: 2;
-  background-color: inherit;
+  background-color: var(--primary-background-color);
```

Comment:
need to check how this one behaves with the sticky - on scrolling horizontally and vertically, and hovering a row or a table header cell, doesn't it causing issues?

Reply:
I checked on all scroll and hover states and it seems to be ok